Google Style Guide page updated

Fsquared123 opened the pull request:
@Nickyshe, 
1. I have downloaded the image and saved it on the blog/img folder
2. I also added a description to the image

Nickyshe commented on blog/google-style-guide.mdx:
```diff
@@ -7,9 +7,10 @@ date: 2024-02-20
-{/* truncate */}
+![Technical writing steps](./img/Fig-1-blog-google-style-guide.png)
 
+_**Figure 1**. Google writing Style Guide_
```
```suggestion
```

Nickyshe replied:
- The image is the cover photo for the article, there is no need to give it a figure caption
-  Rename the image file to `blog-google-style-guide.png`

Nickyshe reviewed (changes requested):
Well done Felicia!!
Here are some few suggestions

![2025-06-06 03_06_25-How to Use a Google Writing Style Guide to Write Performing Articles _ Technical](https://github.com/user-attachments/assets/ad52deed-8f5f-4a80-9d3e-ca7951006eff)
- In this section there is no logical flow. Kindly fix it and remove line spacing 
![2025-06-06 03_11_00-How to Use a Google Writing Style Guide to Write Performing Articles _ Technical](https://github.com/user-attachments/assets/d7b7a566-a58b-4990-ae33-41fb3561bf48)
- Embed the link to the text properly
- Change `Head over` to `Go to`
-  Change `Navigate through` to `Read through`

- Lastly, carefully read in between the lines to fix grammatical errors

Nickyshe commented on blog/introduction-to-technical-writing-mentorship-program.mdx:
```diff
@@ -69,7 +69,7 @@ Their success stories fuel our passion for nurturing the next generation of tech
 As we celebrate three years of empowering beginners, I invite you to join our community.
 
-Whether you're a seasoned technical writer or starting, there's always room to learn, grow, and inspire others. Together, let's elevate the art of technical communication! 🚀
+Whether you're a seasoned technical writer or starting, there's always room to learn. You can grow in your skills and expertise. You can also inspire others. Together, let's elevate the art of technical communication! 🚀
```
```suggestion
Whether you're a seasoned technical writer or starting, there's always room to learn. You can grow in your skills , expertise and, also inspire others. Together, let's elevate the art of technical communication! 🚀
```

Nickyshe commented on blog/introduction-to-technical-writing-mentorship-program.mdx:
```diff
@@ -32,7 +32,7 @@ It all began with a simple message on LinkedIn. A connection expressed admiratio
-Around the same time, others reached out to me via Twitter and LinkedIn. They, too, hungered for knowledge and sought mentorship. Inspired by their enthusiasm, I created a program to empower beginners and provide them with the tools to excel in technical writing.
+Around the same time, others reached out to me via Twitter and LinkedIn. They, too, hungered for knowledge and sought mentorship. Inspired by their enthusiasm, I developed a program to empower beginners. The program equips them with tools to excel in technical writing.
 
 ## Program Structure
```
Format the subheadings in this section properly. The parent heading is ##. The subheading should follow the hierarchy which is ###

Nickyshe commented on docs/technical-writing-course/Writing-for-User-Experience.mdx:
```diff
@@ -56,7 +56,7 @@ The second is this: _writing for user experience and UX writing are the same_.
 
 
-![Fig 2 - User satisfaction in technical writing](./img/Fig-2-ux-user-satisfaction.jpg)
+![User satisfaction in technical writing](./img/Fig-2-ux-user-satisfaction.jpg)
```
Rename this image according to the file name in the image folder

Fsquared123 commented:
@Nickyshe I have implemented your suggestions.
1. Fixed the line spacing issue on Writng for User experience page
2. Corrected the image name in google style guide blog
3. Linked the link to the correct text in google stlye guide blog
4.  Restructured the logic flow of conversation in google style guide blog
